procedures/katello/release.md.erb

```diff
@@ -26,6 +26,10 @@
 <% end -%>
 - [ ] Check for outdated deprecation warnings in the current and next release with `./tools check-deprecation-warnings configs/katello/<%= short_version %>.yaml`. Follow the instructions in the output of the command. Don't forget to create any Redmine issues needed!
 - [ ] Update [docs.theforeman.org](https://github.com/theforeman/foreman-documentation)
+  - Ensure [release notes](https://github.com/theforeman/foreman-documentation/blob/<%= short_version %>/guides/doc-Release_Notes/katello.adoc] are correct
```

Comment:
This and the next bullet point have the issue that `short_version` is the Katello short version, but the branch is the Foreman short version. That's why @chris1984 couldn't find the redmine_release_notes script.